### PR TITLE
Site Logo: change enqueued CSS file depending on lang. & setti…

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* When someone uses the `SCRIPT_DEBUG` constant, or has their site set to a RTL Language, they should get the right stylesheet enqueued.

#### Testing instructions:

* Add Site Logo support to your theme:
```php
$args = array(
	'header-text' => array(
		'site-title',
		'site-description',
	),
	'size' => 'medium',
);
add_theme_support( 'site-logo', $args );
```
* Go to Appearance Customize, and check what files are enqueued on the page.
* You should see `css/site-logo-control.min.css` being enqueued.
* Go to Settings > General and switch your site language to Hebrew.
* You should see `css/site-logo-control-rtl.min.css` being enqueued.
* Add `define( 'SCRIPT_DEBUG', true );` to `wp-config.php`
* You should see `css/site-logo-control-rtl.css` being enqueued.

#### Proposed changelog entry for your changes:

* Site Logo: ensure that the right stylesheet is loaded depending on your site's language.

### CODE REVIEW

**Review by kraftbj — DISMISSED** [review]
Works as advertised. Note for anyone else testing, Jetpack will defer to the core custom logo option. If testing on a theme like Twenty Nineteen, you'd want to delete that section of functions.php OR use another theme that does not support core's custom logo.

**Comment by creativecoder** [comment]
Given I'm hoping to add this file to Fusion (D33578-code) to keep as much of the `site-logo` module in sync with wpcom as possible, will these changes work with wpcom?

**Review by creativecoder — COMMENTED** [review]
Noting that there's something not quite right with the rtl styles here (See how "No logo set" is not centered in the rtl version). I think they need to be regenerated because the rtl version is not complete right now.

Screen Shot 2019-10-10 at 17 51 31
Screen Shot 2019-10-10 at 17 51 54

**Comment by jeherve** [comment]
@creativecoder Good catch indeed! It looks like those had never been updated. I just took care of it in f0722850fdeb4aa61152ce8915d51367a867eb07 and f93cd5658381dc730ae7835ef57aad98e35e2aa8. This should be ready for a new review.

File: modules/theme-tools/site-logo/inc/class-site-logo-control.php [hunk 1]
```diff
@@ -54,8 +54,26 @@ public function enqueue() {
 		// Enqueues all needed media resources.
 		wp_enqueue_media();
 
-		// Enqueue our control script and styles.
-		wp_enqueue_style( 'site-logo-control', plugins_url( '../css/site-logo-control.css', __FILE__ ) );
+		/*
+		 * Enqueue our control script and styles.
+		 */
+
+		// We only enqueue a minified version of the file on prod. Jetppack.
```
Comment (creativecoder) [inline modules/theme-tools/site-logo/inc/class-site-logo-control.php]: Small typo--FYI

Comment (jeherve) [reply modules/theme-tools/site-logo/inc/class-site-logo-control.php]: Oops. Fixed, thank you.

**Review by creativecoder — APPROVED** [review]
Now tested and working as expected.

Note that the `.min` version is loading in my local Jetpack environment, even when the `$min` suffix is an empty string (because `SCRIPT_DEBUG` is `true`). But this is happening with a few other modules too, like `widget-conditions`, so I don’t think it's related to these changes.

**Review by kraftbj — APPROVED** [review]

**Pull request merged**